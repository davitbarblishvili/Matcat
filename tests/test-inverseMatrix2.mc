func main() int 
{
    matrix a; 
    matrix c; 
    a = [[2,5,1],[5,1,6],[3,1,2]];
    c = inv(a);

    printm(c);
    
    return 0;

}