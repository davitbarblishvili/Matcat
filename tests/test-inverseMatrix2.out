(
[-0.12 -0.26 0.85 ]
[0.24 0.03 -0.21 ]
[0.06 0.38 -0.68 ]
)